### PR TITLE
feat: explicit version number and testing

### PR DESCRIPTION
**Pull request by alaterre**

### CODE REVIEW

File: jumanji/registration.py [hunk 1]
```diff
@@ -61,10 +65,7 @@ def get_env_id(name: str, version: Optional[int] = None) -> str:
     Returns:
         The environment ID.
     """
-    version = version or 0
-    full_name = name + f"-v{version}"
-
-    return full_name
+    return name + f"-v{version}"
```
Comment (clement-bonnet) [inline jumanji/registration.py:68]: I'm wondering. Should we check that `version` is an int here?

File: jumanji/registration.py [hunk 2]
```diff
@@ -123,7 +131,7 @@ def register(
     """Register an environment.
 
     Args:
-        id: environment ID, formatted as `(env_name)[-v(version)]`.
+        id: environment ID, formatted as `(env_name)-v(version)`.
```
Comment (clement-bonnet) [inline jumanji/registration.py:134]: NIT
```suggestion
        id: environment ID, formatted as `<env_name>-v<version>`.
```

**Review by clement-bonnet — DISMISSED** [review]
Thank you for the contribution

File: jumanji/registration.py (hunk 1 above)
Comment (alaterre) [reply jumanji/registration.py:68]: The docstring says it must be an `int`. If the user doesn't respect the pre-condition then no guarantee. In any case, this function is only used internally and is not exposed to the user.

**Pull request merged**